content/en-us/reference/engine/classes/VoiceChatService.yaml

```diff
@@ -40,8 +42,24 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: VoiceChatService.UseAudioApi
-    summary: ''
-    description: ''
+    summary: |
+      Controls whether voice chat is implemented using the Audio API.
+    description: |
+      Controls whether voice chat is implemented using the Audio API.
```

Comment:
Does this control whether `AudioDeviceInput`s work at all? Or just whether the default setup consists of the `AudioDeviceInput` chain?

Reply:
Maybe we can summarize the relationship between `UseAudioApi` and `EnableDefaultVoice` more thoroughly, before diving into what exactly happens for each configuration of the two

Reply:
This controls whether AudioDeviceInputs work at all